packages/scans/src/index.js

```diff
@@ -44,9 +44,9 @@ let actionCommon = {
         });
 
```

Comment:
It's better to upload the artifacts and exit the function after LOC 12, as the further down the files, it will try to generate the difference between the current run and the previous run.

Therefore:
```
LOC: 12
if (allowIssueWriting) {
  actionHelper.uploadArtifacts(workSpace, mdReportName, jsonReportName, htmlReportName);
  return;
}

....
let repo;
let tmp = repoName.split('/');
owner = tmp[0];
repo = tmp[1];

```

Reply:
Sure, I can push up a commit in a sec that does it this way.